app/src/components/widgets/singleLevelWidgets/charts/failedCasesTrendChart/failedCasesTrendChart.jsx

```diff
@@ -86,12 +117,22 @@ export class FailedCasesTrendChart extends Component {
     };
 
     this.config = getConfig(params);
+    this.config.data.onclick = this.onChartClick;
```

Comment:
Please add it to `params` object and handle in `getConfig` function to provide more common interface.